src/main/java/hu/bme/mit/spaceship/GT4500.java

```diff
@@ -17,6 +17,7 @@ public GT4500() {
 
   public boolean fireLaser(FiringMode firingMode) {
     // TODO not implemented yet
+    System.out.println("Not implemented yet.");
```

Comment:
Cool, cool, but please actually implement it

Reply:
No

Reply:
Ok, I did them, please check